.github/workflows/nightly.yml

```diff
@@ -233,7 +233,7 @@ jobs:
             -d opcache.jit=1205
       - name: Verify generated files are up to date
         uses: ./.github/actions/verify-generated-files
-  COVERAGE_DEBUG_NTS:
+  COVERAGE_X86_64_DEBUG_NTS:
```

Comment:
I guess more consistent would be `LINUX_X86_64_DEBUG_NTS_COVERAGE`. Or we could drop everything and just do `COVERAGE`, because the arch and configs are not very relevant for these builds. Same with the jobs below.

Reply:
Indeed, as far as coverage is concerned, it has little to do with architecture, so I'll take it back.